clang/lib/Frontend/CompilerInvocation.cpp

```diff
@@ -3218,6 +3218,21 @@ static bool ParseHeaderSearchArgs(HeaderSearchOptions &Opts, ArgList &Args,
   bool IsIndexHeaderMap = false;
   bool IsSysrootSpecified =
       Args.hasArg(OPT__sysroot_EQ) || Args.hasArg(OPT_isysroot);
+
+  // Expand a leading `=` to the sysroot if one was passed (and it's not a
+  // framework flag).
+  auto ConvertHeaderPath = [IsSysrootSpecified,
```

Comment:
I don't love this name. Perhaps `ExpandHeaderSysroot`?

Reply:
I think that `Prefix` or `Expand` would both be better than `Convert`. We aren't really converting this.